nservicebus/msmq/sender-side-distribution-with-distributor.include.md

```diff
@@ -0,0 +1 @@
+INFO: When using-sender side distribution in combination with the distributor delayed retries and delayed deliveries will be routed to the distributor even when the messages where initially sent from client using sender side distribution.
```

Comment:
When using `sender-side` distribution in combination with the distributor `,` delayed retries and delayed deliveries will be routed to the distributor even when the messages where initially sent from client`s` using sender side distribution.

I'm not really sure we need to mention sender-side distribution at all here, as it doesn't matter whether it's just a normal endpoint not going via distributor or a sender using SSD?